### PR TITLE
Fixes MissingDocRuleConfiguration

### PR DESCRIPTION
**Pull request by bdfox325**

### CODE REVIEW

**Comment by bdfox325** [comment]
@jpsim Sorry about that. Created a bug in my previous PR when trying to clean up the code.

**Comment by jpsim** [comment]
Can you cover this with a test?

**Comment by jpsim** [comment]
Can you also add tests exercising the actual rule with these configuration options?

File: Package.resolved [hunk 1]
```diff
@@ -1,6 +1,15 @@
 {
   "object": {
     "pins": [
+      {
```
Comment (bdfox325) [inline Package.resolved:4]: This was generated when running `make docker_test`

Comment (jpsim) [reply Package.resolved:4]: Please revert, as CyptoSwift is only used on Linux.

**Comment by jpsim** [comment]
Thanks for the fix!

**Pull request merged**